src/swarm/neo/authentication/CredentialsFile.d

```diff
@@ -187,11 +188,11 @@ private HmacDef.Key[istring] parseContent ( istring content,
             b |= hexToByte(hex_key[i++]);
         }
 
-        assert(i == hex_key.length);
+        verify(i == hex_key.length);
```

Comment:
This is already checked by `enforce` in line 163 so it can keep being `assert`.

Reply:
Updated.